src/Core/ServiceWrapper/Program.cs

```diff
@@ -124,67 +142,55 @@ public static void Run(string[] argsArray, ServiceDescriptor? descriptor = null)
                 elevated = IsProcessElevated();
             }
 
-            switch (args[0].ToLower())
+            switch (obj)
```

Comment:
💭 Is there no better way than using `switch`? Maybe polymorphism.

Reply:
Would be great. But isn't it reduce the readability?

Reply:
Something that I can guess is method overloading. But I think it will reduce the code readability?

Reply:
Why does method overloading reduce the readability? Why was OOP invented?

Reply:
Method overriding.

Reply:
Take local functions of commands (`install`, `start` . . . ) into particular CLI utility class with a common name like `RunCommand` and call that method inside the `Run` method?

Reply:
https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/polymorphism